### PR TITLE
Feature/avo 2205

### PR DESCRIPTION
**Pull request by WardVerc1**
https://meemoo.atlassian.net/browse/AVO-2205

### CODE REVIEW

**Comment by bertyhell** [comment]
I made a few tweaks to the styling to better match the other tables and modals in avo:

before:
image
image

after:
image
image

**Review by bertyhell — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -30,12 +30,13 @@
   "dependencies": {
     "@flowplayer/player": "3.2.8",
     "@hookform/resolvers": "^2.9.9",
+    "@meemoo/admin-core-ui": "v2.0.8",
```
Comment (ian-emsens-shd) [inline package.json:33]: Why downgrade this dependency?

File: src/admin/translations/views/TranslationsOverviewV2.module.scss [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+			width: 100%;
+			appearance: none;
+			transition: all 0.15s ease-in-out;
+
+			&:focus {
+				outline: 0;
+				border-color: #25a4cf;
+				box-shadow: inset 0 0 0 0.1rem #25a4cf;
+			}
+		}
```
Comment (ian-emsens-shd) [inline src/admin/translations/views/TranslationsOverviewV2.module.scss:40]: Most of these colors are probably available in the variables.

edit: check other values for occurence in variables as well

**Review by ian-emsens-shd — CHANGES_REQUESTED** [review]

File: src/admin/translations/views/TranslationsOverviewV2.module.scss (hunk 2 above)
Comment (bertyhell) [reply src/admin/translations/views/TranslationsOverviewV2.module.scss]: fixed

**Pull request merged**